examples/util.h

```diff
@@ -2,12 +2,20 @@
 
 #include <opentimelineio/errorStatus.h>
 
+#include <vector>
+
 // Normalize path (change '\' path delimeters to '/').
 std::string normalize_path(std::string const&);
 
+// Get the absolute path.
+std::string absolute(std::string const&);
+
 // Create a temporary directory.
 std::string create_temp_dir();
 
+// Get a list of files from a directory.
+std::vector<std::string> glob(std::string const& path, std::string const& pattern);
+
 // Print an error to std::cerr.
 void print_error(opentimelineio::OPENTIMELINEIO_VERSION::ErrorStatus const&);
```

Comment:
I think it might help for locating this code to put it into an `example` or `util` namespace-  I see you left the otio code in the `otio` namespace, which is great, and to make it super obvious to folks who are only scanning the code on GitHub where these helper functions come from, I think throwing them in a namespace as well helps.

Reply:
That's a good idea, I like ```example``` just so it's obvious this is not part of the actual otio API.